src/lib/viewers/doc/_docBase.scss

```diff
@@ -332,7 +332,27 @@ $thumbnail-sidebar-width: 226px;
     }
 }
 
-// Hide the toggle thumbnails button when in fullscreen
-.bp-content.bp-is-fullscreen .bp-toggle-thumbnails-icon {
-    display: none;
+.bp-content {
+    &.bp-annotations-discoverable {
+        .textLayer {
```

Comment:
Can you also see if we need to mention the pdfjs annotationLayer as well?

Reply:
Yep, we do need it. Added a fix.